lib/install/config/loaders/installers/elm.js

```diff
@@ -4,11 +4,11 @@ const { env } = require('../configuration.js')
 const elmSource = path.resolve(process.cwd())
 
 const loaderOptions = () => {
-    if (env.NODE_ENV === "production") {
-        return `elm-webpack-loader?cwd=${elmSource}`
-    }
+  if (env.NODE_ENV === 'production') {
```

Comment:
The current revisions is the mirror image of that - if you wanted debug-compilation in your staging environment for example, you could say `NODE_ENV=development ./bin/webpack`.

To fully support various scenarios people might have, I think we'd need to expose an env variable allowing people to customize the loader/compiler flags. I didn't want to go down that road yet, in hopes that this PR will provide reasonable defaults without additional user config options.